state-chain/chains/src/btc/api.rs

```diff
@@ -17,11 +17,41 @@ pub enum BitcoinApi<Environment: 'static> {
 	_Phantom(PhantomData<Environment>, Never),
 }
 
+/// A Bitcoin exclusive Api call, used to rotate Utxos from the previous Vault into the current
+/// vault.
+pub trait TransferUtxoCall: ApiCall<BitcoinCrypto> {
+	fn transfer_utxo(
+		old_key: [u8; 32],
+		new_key: [u8; 32],
+		utxos: Vec<Utxo>,
+		change_amount: BtcAmount,
+	) -> Self;
+}
+
+impl<E> TransferUtxoCall for BitcoinApi<E> {
+	fn transfer_utxo(
+		old_key: [u8; 32],
+		new_key: [u8; 32],
+		utxos: Vec<Utxo>,
+		change_amount: BtcAmount,
+	) -> Self {
+		// We will use the bitcoin address derived with the salt of 0 as the vault address.
+		let new_vault_change_script =
```

Comment:
Also see here, this is very similar to the consolidation call - it seems like these can just be merged into one call. Both calls are already key agnostic. Sending from old to new. This would help making these two btc automatic send code paths into one.